tests/test_csc.py

```diff
@@ -86,11 +86,11 @@ async def test_configuration(self):
                 with self.subTest(bad_config_name=bad_config_name):
                     with salobj.assertRaisesAckError():
                         await self.remote.cmd_start.set_start(
-                            settingsToApply=bad_config_name, timeout=STD_TIMEOUT
+                            override=bad_config_name, timeout=STD_TIMEOUT
                         )
 
             await self.remote.cmd_start.set_start(
-                settingsToApply="all_fields", timeout=STD_TIMEOUT
+                override="all_fields", timeout=STD_TIMEOUT
```

Comment:
All your override="x" must include the `.yaml` suffix, e.g. `override="all_fields.yaml"`

The old format worked because we had labels, but those are gone in the new configuration system.